fate-manager/src/services/component_deploy_service/component_deploy_service.go

```diff
@@ -152,19 +154,15 @@ func GetLog(logReq entity.LogReq) (map[string][]string, error) {
 	if err != nil || len(deploySiteList) == 0 {
 		return nil, err
 	}
-	cmd := "kubectl get pods -n kube-fate | grep kubefate|grep Running| awk '{print $1}'"
-	if setting.KubenetesSetting.SudoTag {
-		cmd = fmt.Sprintf("sudo %s", cmd)
-	}
-	result, _ := util.ExecCommand(cmd)
-	if len(result) == 0 {
-		return nil, err
+	result, err := clientgo.ClientSet.GetPodWithPattern("kube-fate", "kubefate")
```

Comment:
Not a good idea to hard code here. At least we set it to const or allow to pass from outside.